services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -201,9 +201,9 @@ const Asset = ({ libraryData }) => {
                       <dd className={dashboardStyles.meta}>
                         <StatusIcon
                           className={styles['status-icon']}
-                          status={assetData.content.status}
+                          status={assetData.statusKey}
                         />
-                        {get(status, `[${assetData.content.status}].name`, '–')}
+                        {get(status, `[${assetData.statusKey}].name`, '–')}
```

Comment:
I haven't seen this syntax before and had to look up what this does in the lodash docs. What's the advantage of doing this as opposed to `status[assetData.statusKey].name`?

Reply:
in this particular case, that if the result of `status[assetData.statusKey].name` is undefined or null, it will default to '-' .

Also lodash won't error out if status[assetData.statuskey] is undefined and the other approach will

Reply:
Hmm. What about doing it with optional chaining to avoid the lodash overhead?

`status?.[assetData.statusKey]?.name || '-'`